src/algorithms/common.jl

```diff
@@ -239,3 +239,28 @@ function optimize(model::AbstractModel, optimizer::AbstractOptimizer, x0, args..
     r = optimize(_model, optimizer, _x0, args...; kwargs...)
     return @set r.minimizer = unflatten(r.minimizer)
 end
+
+"""
+ optimize without x0
+"""
+ function optimize(model::AbstractModel, optimizer::AbstractOptimizer, args...; kwargs...)
```

Comment:
Why do we need this?

Reply:
Original algorithm in paper doesn't have x0, they sample a simulated orthogonal array, I'm trying to fit that. Also some multistart algorithm doesn't need x0 I think they can use this interface.